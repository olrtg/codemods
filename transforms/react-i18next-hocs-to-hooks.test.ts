import { defineSnapshotTest } from 'jscodeshift/src/testUtils'
import transformer from './react-i18next-hocs-to-hooks'

describe('react-i18next HOCs to hooks', () => {
  defineSnapshotTest(
    transformer,
    {},
    "import { withTranslation } from 'react-i18next'",
  )
})
