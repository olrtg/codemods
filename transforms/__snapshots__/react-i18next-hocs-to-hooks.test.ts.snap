// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-i18next HOCs to hooks transforms correctly 1`] = `"import { withTranslation } from 'react-i18next'"`;
